charm/docs/reference/github-auth.md

```diff
@@ -0,0 +1,30 @@
+# GitHub Authentication
+
+This section describes the GitHub authentication options available for the charm.
+
+You can either choose to use
+
+- classic personal access tokens (PATs)
+- fine-grained access tokens
+- a GitHub app
+
+for authentication. The latter two options are recommended for better security and access control.
+They require the fine-grained permissions as mentioned below.
+
+
+## Personal access token scopes
+
+If you want to use classic PATS, you will need to select the `repo` scope.
```

Comment:
For my understanding: How and where do you select the `repo` scope?

Reply:
Hi Erin, when you create a classic PAT in the UI, you can select the scope. See the attached screenshot .
![image](https://github.com/user-attachments/assets/b4304a35-212f-4a67-b399-077e29191463)

Reply:
That makes sense, thank you so much!

One other question about this line: Does PATS stand for "Personal access token scopes" here, or is it the same PATs definition previously mentioned?

Reply:
PAT stands for Personal Access Token. PATS should be the plural, but I probably should not have used a capital S.

Reply:
Thank you for the clarification. I agree that a capital S should not be used :)

Reply:
Thanks for finding this. I have updated the docs.